doc/release-notes/release-7.sgml.in

```diff
@@ -60,6 +60,10 @@ This section gives an account of those changes in three categories:
 	<tag>buffered_logs</tag>
 	<p>Honor the <em>off</em> setting in 'udp' access_log module.
 
+	<tag>cachemgr_passwd</tag>
+	<p>Removed the <em>non_peers</em> action. See the Cache Manager
```

Comment:
> IMO the fix would be to make the list a cache with TTL expiry

Yes, if this feature is needed, then it should be implemented using a dedicated record type for unwanted responses from a given source address and a limited-capacity cache of those records (probably a ClpMap).

> I am not going to object to removal in v7

You made the right call IMO: The current implementation is badly broken, on several levels, and is in the way of important in-progress improvements. We do not know of any active users of these rather unusual/special cache manager reports. If somebody needs this reporting feature, it should be implemented correctly, guided by the real use cases. This removal does not preclude such future implementation or make it any more difficult. In fact, I think it makes it easier!